src/sequence.lisp

```diff
@@ -61,6 +61,31 @@
              (let ((,elt (aref ,nseq ,index)))
                ,@body))))))
 
+(defun count (item sequence &key from-end (start 0) end
+                                 key (test #'eql testp)
+                                 (test-not #'eql test-not-p))
+  ;; TODO: Implement START and END efficiently for all the sequence
+  ;; functions.
+  (let* ((l (length sequence))
```

Comment:
using a more descriptive name than l would be better. Maybe sequence-length or even length.